src/utils/feathers-mongoose/service.js

```diff
@@ -462,6 +462,17 @@ class Service extends AdapterBase {
 
 	filterQuery(params) {
 		const options = this.getOptions(params);
+
+		// $limit=false - should return all records with structure as with pagination (total, limit, skip, data)
+		const paginateNoLimit = {
+			default: undefined,
+			max: undefined,
+		};
+		if (params.query && (params.query.$limit === 'false' || params.query.$limit === false)) {
+			options.paginate = paginateNoLimit;
```

Comment:
add a log here to know what query caused this so we know what was affected after all?

Reply:
there are maybe 20 places in code which still use $limit: false